src/lib/data/alignment.js

```diff
@@ -790,6 +800,112 @@ export default class Alignment {
     return true
   }
 
+  /**
+   * Adds a line break after the token
+   * @param {Token} token
+   * @returns {Boolean}
+   */
+  addLineBreakAfterToken (token) {
+    const segment = this.getSegmentByToken(token)
+    const alignedText = this.getAlignedTextByToken(token)
+
+    const newIdWord = alignedText.getNewIdWord({
+      token,
+      segment,
+      changeType: TokensEditController.changeType.LINE_BREAK
+    })
+
+    token.addLineBreakAfter()
+    token.update({
+      idWord: newIdWord
+    })
+    return true
+  }
+
+  /**
+   * Moves the token to the next/previous segment
+   * @param {Token} token
+   * @param {TokensEditController.direction} direction
+   * @returns {Boolean}
+   */
+  moveToSegment (token, direction) {
+    const segment = this.getSegmentByToken(token)
+    const newSegment = (direction === TokensEditController.direction.PREV) ? this.getPrevSegmentByToken(token) : this.getNextSegmentByToken(token)
+
+    const tokenIndex = segment.getTokenIndex(token)
+    segment.deleteToken(tokenIndex)
+
+    const alignedText = this.getAlignedTextByToken(token)
+    const newIdWord = alignedText.getNewIdWord({
+      token,
+      segment: newSegment,
+      changeType: (direction === TokensEditController.direction.PREV) ? TokensEditController.changeType.TO_PREV_SEGMENT : TokensEditController.changeType.TO_NEXT_SEGMENT
+    })
+
+    token.update({
+      idWord: newIdWord,
+      segmentIndex: newSegment.index
+    })
+
+    const insertPosition = (direction === TokensEditController.direction.PREV) ? newSegment.tokens.length : 0
+    newSegment.insertToken(token, insertPosition)
+    return true
+  }
+
+  /**
+   * Check if the token could be merged with the previous
+   * @param {Token} token
+   * @returns {Boolean}
+   */
+  allowedMergePrev (token) {
+    return Boolean(this.getPrevToken(token))
+  }
+
+  /**
+   * Check if the token could be merged with the next
+   * @param {Token} token
+   * @returns {Boolean}
+   */
+  allowedMergeNext (token) {
+    return Boolean(this.getNextToken(token))
+  }
+
+  /**
+   * Check if the token could be splitted (for now it is always true)
+   * @param {Token} token
+   * @returns {Boolean}
+   */
+  allowedSplit (token) {
+    return true

```

Comment:
if the token contains only a single character we should not allow split

Reply:
Oh thank you, didn't guess to this. That's why created it for future ideas/requirements for consistency.

Reply:
done

Reply:
I didn't think of it either until I saw your method that was waiting for it :-)